optimize/backend/src/main/java/io/camunda/optimize/IgnoreDuringScan.java

```diff
@@ -18,6 +18,7 @@
  * @ComponentScan(excludeFilters = @ComponentScan.Filter(IgnoreDuringScan.class))
  * @Configuration
  * public class Main {
+ * }
```

Comment:
This was giving compilation error: even though it's in a comment block, it didn't close a docstrings tag well